olympia.py

```diff
@@ -16,13 +15,8 @@ def update_system_path():
     are importable without the `apps.` prefix."""
 
     ROOT = os.path.dirname(os.path.abspath(__file__))
-
-    prev_sys_path = set(sys.path)
-
-    site.addsitedir(os.path.join(ROOT, 'apps'))
-
-    # Move the new items to the front of sys.path.
-    sys.path.sort(key=prev_sys_path.__contains__)
+    # Insert the 'apps' folder to the front of sys.path so it takes precedence
```

Comment:
Please end the comment with a `.`: we word our comments to be sentences (they start with an uppercase, and end with a dot.